backends/bmv2/common/options.h

```diff
@@ -37,6 +39,9 @@ class BMV2Options : public CompilerOptions {
         registerOption("-o", "outfile",
                 [this](const char* arg) { outputFile = arg; return true; },
                 "Write output to outfile");
+        registerOption("--fromJSON", "file",
+                [this](const char* arg) { loadIRFromJson = true; file = arg; return true; },
+                "Use IR representation from JsonFile dumped previously.");
```

Comment:
The message should probably indicate that compilation starts after mid-end in this case.